formula/src/main/java/com/instacart/formula/Listener.kt

```diff
@@ -0,0 +1,13 @@
+package com.instacart.formula
+
+/**
+ * Formula uses Listener callbacks to pass events.
+ */
+fun interface Listener<Event> : (Event) -> Unit {
```

Comment:
Still debating what would be the right name for this.

Reply:
I prefer `Listener`

Reply:
Another option would be `Handler`. I don't have a strong preference, just throwing it out there to see what other people think.

Reply:
Yeah, I've also thought about `Handler` though I didn't like its associations with `android.Handler`